manifests/c/CollapseLauncher/Collapse/1.82.9/CollapseLauncher.Collapse.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: CollapseLauncher.Collapse
+PackageVersion: 1.82.9
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-12-25

```

Comment:
You may need to add MinimumOSVersion to the installer manifest, but I'm not sure exactly what MinimumOSVersion is.
From the [document](https://collapselauncher.com/docs/installation.html#how-do-i-know-if-i-can-run-collapse):
![image](https://github.com/user-attachments/assets/0b6f02ad-bc5b-4f7f-ab13-498ed3afd61b)